test/nn/test_embedding.py

```diff
@@ -602,7 +602,8 @@ def embedding_bag_check(indices, weights, mode, sparse, padding_idx):
                     # Fill one row with duplicate index so we can test with a fully
                     # padded row
                     duplicate_row = random.randint(0, indices_dim0 - 1)
-                    indices[duplicate_row] = indices[duplicate_row][0]
+                    fill_value = indices[duplicate_row][0].clone()
```

Comment:
Commenting for posterity that this change is because `copy_` uses TensorIterator which does [`assert_no_partial_overlap`](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/MemoryOverlap.cpp#L76-L78) and `fill_` is called in `copy_to` [here](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/TensorIndexing.h#L409-L411)

Reply:
So this was a bc-breaking change?

Reply:
Yea we missed this, Driss later reverted this in https://github.com/pytorch/pytorch/pull/109835